locale/es_ES/locale.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * @file locale/en_US/locale.xml
+  *
+  * Copyright (c) 2003-2018 Simon Fraser University
```

Comment:
@gonetil, could you change the copyright end dates to 2019? Thanks!

Reply:
Done Alec!